.travis.yml

```diff
@@ -19,6 +19,13 @@ sudo: required
 services:
   - docker
 
+install:
+  - . $HOME/.nvm/nvm.sh
+  - nvm install 8.9.0
+  - nvm use 8.9.0
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
This will skip the tests ? of the composer ?

Reply:
@hemikak - Yes, this will skip the tests in the build phase. Tests will be executed using `mvn test` command later.

Reply:
Ah right 👍